test/test_utilities.py

```diff
@@ -150,6 +150,20 @@ def test_redact_credentials():
     assert sql_helpers.redact_credentials(test_str) == test_result
 
 
+def test_get_sql_from_file():
+
+    # Test query string
+    test_str = "select * from schema.tablename limit 10"
+
+    # Create fake file.
+    os.mkdir("tmp")
```

Comment:
The GMail tests are failing because they're also trying to create this directory, which isn't being cleaned up in the test. If we do decide to retain the `get_sql_from_file` method, the test should be rewritten to use the `tempfile` module (https://docs.python.org/3/library/tempfile.html).